lib/multi-acc-new-eks-mixed-observability-pattern/pipeline.ts

```diff
@@ -0,0 +1,304 @@
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { utils } from '@aws-quickstart/eks-blueprints';
+import * as cdk from 'aws-cdk-lib';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { Construct } from 'constructs';
+import AmpMonitoringConstruct from './amp-monitoring';
+import CloudWatchMonitoringConstruct from './cloudwatch-monitoring';
+import GrafanaOperatorConstruct from "./grafana-operator-index";
+import { AmgIamSetupStack, AmgIamSetupStackProps } from './amg-iam-setup';
+import { getAMPAccessPolicyDocument } from './amp-access-policy';
+import { getCWAccessPolicyDocument } from './cw-access-policy';
+import { getCodeBuildPolicyDocument } from './codebuild-policy';
+import { CreateIAMRoleNestedStack, CreateIAMRoleNestedStackProps } from './create-iam-role';
+import { getSSMSecureString } from './get-ssm-securestring';
+
+const logger = blueprints.utils.logger;
+
+let ampAccount: string;
```

Comment:
remove these variable definits and add it while being used

Reply:
I am declaring them upfront as they are used in class `PipelineMultiEnvMonitoring` and in function createGOArgoAddonConfig